docs/orms.rst

```diff
@@ -322,6 +322,22 @@ To work, this class needs an `SQLAlchemy`_ session object affected to the :attr:
     .. attribute:: force_flush
 
         Force a session flush() at the end of :func:`~factory.alchemy.SQLAlchemyModelFactory._create()`.
+        
+        This option is deprecated. Use ``sqlalchemy_session_persistence`` instead.
+    
+    .. attribute:: sqlalchemy_session_persistence
```

Comment:
Can you move this attribute above the `force_flush` attribute? I know it's not alphabetical, but I want someone reading the docs to see this before the deprecated one.